src/python/grpcio_tests/tests/unit/framework/interfaces/face/_future_invocation_asynchronous_event_service.py

```diff
@@ -450,6 +452,7 @@ def testFailedUnaryRequestUnaryResponse(self):
           with self.assertRaises(face.ExpirationError):
             response_future.result()
           self.assertIsNotNone(response_future.traceback())
+          self.assertTrue(abortion_callback._called)
```

Comment:
Perhaps I'm not reading it correctly, but shouldn't `abortion_callback.future()` only loop until `abortion_callback` itself is merely called? Not necessarily called-and-passed-a-future?